File: C++/Cristian Maldonado/Grafos/Matriz_De_Adyacencia.cpp
#include<bits/stdc++.h>
#include<cstdlib>
#define MAX 70

using namespace std;

static int G[MAX][MAX];

int main (int argc, char *argv[]) {
	memset(G, 0, sizeof G);
	int vertices = 0, caminos = 0;
	scanf("%i %i", &vertices, &caminos);
	for(int i = 0; i < caminos; i++){
		int desde = 0, hasta = 0, peso = 0;
		scanf("%i %i %i", &desde, &hasta, &peso);
		G[desde][hasta] = peso;
		G[hasta][desde] = peso;
	}
	cout<<""<<endl;
	for(int i = 0; i < vertices; i++){
		for(int j = 0; j < vertices; j++){
				printf("%i\t",G[i][j]);
		}
		cout<<""<<endl;
	}
	return 0;
}

File: Java/Cristian Maldonado/Grafos/Matriz_De_Adyacencia.java
import java.util.Arrays;
import java.util.Scanner;

public class Matriz_De_Adyacencia {

    static int G[][];
    static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) {
        int Vertices, Caminos;
        Vertices = sc.nextInt();
        Caminos = sc.nextInt();
        G = new int[Vertices][Vertices];
        for (int i = 0; i < Vertices; i++) {
            Arrays.fill(G[i], Integer.MAX_VALUE);
        }
        for (int i = 0; i < Vertices; i++) {
            G[i][i] = 0;
        }
        for (int i = 0; i < Caminos; i++) {
            int desde, hasta, peso;
            desde = sc.nextInt();
            hasta = sc.nextInt();
            peso = sc.nextInt();
            // Grafo dirigido
            G[desde][hasta] = peso;
            // Complementar con la siguiente linea para no dirigido
            G[hasta][desde] = peso;
        }
        for (int i = 0; i < Vertices; i++) {
            for (int j = 0; j < Vertices; j++) {
                if (G[i][j] == Integer.MAX_VALUE) {
                    System.out.print("inf" + "\t");
                } else {
                    System.out.print(G[i][j] + "\t");
                }
            }
            System.out.println();
        }
    }
}
